content/docs/09.5-security.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Security"
+metaTitle: "Spectro Cloud Security"
+metaDescription: "Get an overview of Palette's security awareness and controls and where yo can report security issues."
```

Comment:
It's not only about Palette security but also about our security culture

Reply:
@cloudmaniac Revised to say, "Get an overview of Palette's security controls, security-aware culture, and where you can report any security issues."